scss/_navbar.scss

```
@@ -183,16 +183,22 @@
   // Boosted mod: Handle .nav-link and .nav-icon slightly differently than Bs
   .nav-link {
     position: relative;
-    padding-top: subtract(var(--#{$prefix}navbar-nav-link-padding-y), var(--#{$prefix}navbar-border-width));
-    margin-left: calc(-1 * var(--#{$prefix}navbar-nav-link-padding-x)); /* stylelint-disable-line function-disallowed-list */
+    margin-top: subtract(var(--#{$prefix}nav-link-margin-y), var(--#{$prefix}navbar-border-width));
+    margin-left: 0;
     word-break: break-all;
 
-    &:hover {
-      z-index: 1;
+    &::before {
+      position: absolute;
+      top: calc(-1 * var(--#{$prefix}nav-link-margin-y, 5px)); // stylelint-disable-line function-disallowed-list
+      right: calc(-1 * var(--#{$prefix}nav-link-margin-x, 0px)); // stylelint-disable-line function-disallowed-list
+      bottom: calc(-1 * var(--#{$prefix}nav-link-margin-y, 5px)); // stylelint-disable-line function-disallowed-list
+      left: calc(-1 * var(--#{$prefix}nav-link-margin-x, 0px)); // stylelint-disable-line function-disallowed-list
```

Not sure but may be better 🤔 

```suggestion
/* stylelint-disable function-disallowed-list */
      top: calc(-1 * var(--#{$prefix}nav-link-margin-y, 5px));
      right: calc(-1 * var(--#{$prefix}nav-link-margin-x, 0px));
      bottom: calc(-1 * var(--#{$prefix}nav-link-margin-y, 5px));
      left: calc(-1 * var(--#{$prefix}nav-link-margin-x, 0px));
/* stylelint-enable function-disallowed-list */
```